spec/meilisearch/index/updates_spec.rb

```diff
@@ -67,6 +67,12 @@
     @index.add_documents(@documents)
     response = @index.add_documents(@documents)
     update_id = response['updateId']
+
+    # check if some unknown error occurred
+    expect do
+      @index.wait_for_pending_update(update_id, 1)
+    end.to raise_error(StandardError)
```

Comment:
Hi @Chhekur 

In which case would this call throw a standard error which is not a `MeiliSerch::TimeoutError`? 

I personally haven't seen it, and I'm concerned because I don't think this should happen in the first place, so maybe it is not something we should fix in the tests, but in the actual code (just guessing)!

Can you give some details on, for example, how to reproduce?

Thanks!!!

Reply:
It is possible that some unknown runtime exception occur and in specs there is no expect for such unknown error that's why sometimes test fails.

Yeah we could fix this in code so that we can make sure nothing will break in prod.

Reply:
It's indeed not random exceptions that are thrown but always the same one `Zlib`.
Fix the test is maybe a trick to hide a bug 😕

Reply:
But in this new calld to `wait_for_pending_update` the test is are clearly 'expecting' a `StandardError`. Maybe I didn't really uderstand what we want to fix here. But from my point of view, this spec is actually saying that this `StandardError` *should* be raisen, and if it doesn't it means that something went wrong and test will fail. Which doesn't seem justified by a probability of something else 'probably' failing.

Reply:
Can you please tell me how to reproduce this error. I really prefer to check if it is not coming from a bug in the SDK that should be fixed :)